README.md

```diff
@@ -15,4 +15,67 @@ To install the latest stable version on openSUSE or SLE, use zypper:
     $ sudo zypper install yast2-auth-server
 
 # Run
-Visit Yast control panel and launch "Create New Kerberos Server" or "Create New Directory Server".
\ No newline at end of file
+Visit Yast control panel and launch "Create New Kerberos Server" or "Create New Directory Server".
+
+
+# Development
+
+You need to prepare your environment with:
+
+```
+ruby_version=$(ruby -e "puts RbConfig::CONFIG['ruby_version']")
+zypper install -C "rubygem(ruby:$ruby_version:yast-rake)"
+zypper install -C "rubygem(ruby:$ruby_version:rspec)"
+zypper install git yast2-devtools yast2-testsuite yast
+```
+
+You can then run the auth-server module with:
+
+```
+rake run
+rake run[module name]
+rake run[ldap-server]
+```
+
+For the 389-ds setup, you'll require a CA + pkcs12 bundle with a cert to use. You can generate
+these with certutil from the package mozilla-nss-tools.
+
+```
+mkdir local_ca
+cd local_ca
+echo "password" > password.txt
+certutil -N -f password.txt -d .
+certutil -S -n CAissuer -t "C,C,C" -x -f password.txt -d . -v 24 -g 4096 -Z SHA256 --keyUsage certSigning -2 --nsCertType sslCA -s "CN=ca.nss.dev.example.com,O=Testing,L=example,ST=Queensland,C=AU"
+
+certutil -S -n Server-Cert -t ",," -c CAissuer -f password.txt -d . -s "CN=test_b.dev.example.com,O=Testing,L=example,ST=Queensland,C=AU"
+
+certutil -L -n CAissuer -a -d . > ca.pem
+pk12util -o server-export.p12 -d . -k password.txt -n Server-Cert
+
+/home/admin/local_ca/
+```
```

Comment:
really nice howto. BTW what mean that last line with path? You can add some `# comments` there, so it is still executable.

Reply:
There are some fixes on the certificate inclusion for the ds configuration in an other PR, are these included in this PR or are they irrelevant? https://github.com/yast/yast-auth-server/pull/48

Reply:
Those fixes will partially work, and will partially break. The Server-Cert change will break (basically, 389 expects it to be Server-Cert), but the "pkpass" change would work. So that PR will need a refactor I think in light of this change.

Reply:
So thinking about it, a possible solution is to ask for the ca, cert and key all as .pem, then we do the pkcs12 generation on your behalf to guarantee it's named correctly?